README.md

```diff
@@ -412,7 +412,7 @@ Example commands to run test suite for the dashboard `component` only, with the
 make run-nowebhook
 ```
 ```shell
-make e2e-test -e OPERATOR_NAMESPACE=<namespace> -e E2E_TEST_FLAGS="--test-operator-controller=false --test-webhook=false --test-component=dashboard"
+make e2e-test -e OPERATOR_NAMESPACE=<namespace> -e E2E_TEST_FLAGS="--test-operator-controller=false --test-webhook=false --test-component=dashboard,ray,modelregistry, trainingoperator"
```

Comment:
this does need to be changed, the example is about running a test for a single component (line 409)

Reply:
reverted